project.godot

```diff
@@ -114,6 +114,66 @@ _global_script_classes=[ {
 "language": "GDScript",
 "path": "res://addons/gdUnit3/test/mocker/resources/ClassWithVariables.gd"
 }, {
+"base": "Reference",
```

Comment:
Could we make one the main scene to run projects tests immediately?

```git checkout ...
cd ...
godot
```

Reply:
I don't get the point.
What do you mean?
GdUnit is an plugin and adds a test inspector where is like the main scene.